{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -42,17 +42,14 @@
 
 # SECURITY
 # ------------------------------------------------------------------------------
-# https://docs.djangoproject.com/en/dev/ref/settings/#secure-proxy-ssl-header
-SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+{% if cookiecutter.use_docker -%}
+# NOTE headers are managed by the security-headers middleware in traefik.yml
```

Comment:
I wonder if we should comment the code out instead of removing these entirely...? For example, I deploy on AWS using Docker, so I would like to have these headers.

Reply:
That seems like a good compromise to me, maybe a note to say these settings won't take effect if using traefik

Reply:
Cool. Something like "Uncomment if not using Traefik"